src/routes/AccountDetailsContainer.js

```diff
@@ -126,7 +127,14 @@ class AccountDetailsContainer extends React.Component {
     const patronGroup = this.getPatronGroup();
     const itemDetails = this.getItemDetails();
 
-    if (!account) return (<ViewLoading defaultWidth="100%" paneTitle="Loading accounts" />);
+    if (!account) {
+      return (
+        <LoadingView
+          defaultWidth="100%"
+          paneTitle={<FormattedMessage id="accounts.loading" />}
```

Comment:
Believe all of. these need to be
```suggestion
          paneTitle={<FormattedMessage id="ui-users.accounts.loading" />}
```